Supplier Landing Page implemented

DariusV72 opened the pull request.

DariusV72 committed:
Supplier Landing Page implemented

CimpeanAndreea commented on lib/supplierlandingpage/BarChart/BarChart.dart:
```diff
@@ -0,0 +1,75 @@
+    Colors.orangeAccent,
+  ];
+
+  //const BarChartExist(BarChartData barChartData, {Key? key}) : super(key: key);
```
Remove the comment

CimpeanAndreea commented on lib/supplierlandingpage/BarChart/BarChartImproved.dart:
```diff
@@ -0,0 +1,212 @@
+        show: false,
+      );
+
+  final _barsColor = Color.fromARGB(255, 235, 130, 11);
```
Define constant for color in constants file

CimpeanAndreea commented on lib/supplierlandingpage/pieChart.dart:
```diff
@@ -0,0 +1,47 @@
+                  padding: const EdgeInsets.fromLTRB(35, 15, 25, 15),
+                  child: Container(
+                    constraints: BoxConstraints(maxHeight: 25, maxWidth: 25),
+                    //decoration: BoxDecoration(shape: BoxShape.circle),
```
Remove the comment if it doesn't describe something useful to understanding the code or it is not intended to be changed to this in future

CimpeanAndreea commented on lib/supplierlandingpage/pieChart.dart:
```diff
@@ -0,0 +1,47 @@
+  @override
+  Widget build(BuildContext context) => Container(
+        child: Row(
+            //mainAxisAlignment: MainAxisAlignment.spaceEvenly,
```
Remove comment

CimpeanAndreea commented on lib/supplierlandingpage/main.dart:
```diff
@@ -0,0 +1,27 @@
+  runApp(const MyApp());
+}
+
+class MyApp extends StatefulWidget {
```
Don't we already have such a Widget "Main"? Why do you have this?

CimpeanAndreea commented on lib/supplierlandingpage/constants.dart:
```diff
@@ -0,0 +1,28 @@
+const titleFontSize = 38.0;
```
Why do you have your own constants file? It should be just 1 single file for the entire project, unless it contains only constants that are really necessary just for your widget, but I doubt it, since here you have things related to register. Is this a mistaken duplicate of the original constants file?

CimpeanAndreea commented on lib/supplierlandingpage/PieChart_Data/pieChart_sectionData.dart:
```diff
@@ -0,0 +1,29 @@
+        value: data.percent,
+        title: '',
+        radius: 15,
+        //titleStyle: TextStyle(
```
Remove comments if they are not necessary

CimpeanAndreea commented on lib/supplierlandingpage/BarChart/BarChartImproved.dart:
```diff
@@ -0,0 +1,212 @@
+
+  Widget getTitles(double value, TitleMeta meta) {
+    const style = TextStyle(
+      color: Color(0xff7589a2),
```
Create constant for color

CimpeanAndreea commented on lib/supplierlandingpage/BarChart/BarChart.dart:
```diff
@@ -0,0 +1,75 @@
+
+  @override
+  Widget build(BuildContext context) {
+    // TODO: implement build
```
Is this not the build method from the "TODO" comment? :D

CimpeanAndreea reviewed (commented):
Please rename all files and folders that do not follow already existing naming conventions. For example "pieChart_data" seems to be a combination of patterns, check to see how the other files in the project are named using camel style/dashes. Also remove all comments that are no longer useful.

DariusV72 committed:
Fixed issues with the supplier landing page

Removed unnecessary comments as well as the unneeded main and constants file in the supplierlanding folder. Modified file names to match with the rest of the project

DariusV72 committed:
Missed a few colors when making constants, fixed now

DariusV72 committed:
Fixed issues with supplier landing page

DariusV72 committed:
Delete main.dart

DariusV72 committed:
Merge branch 'master' into supplier_landing_page

DariusV72 committed:
Few changes have been made, changed app bar logo

DariusV72 committed:
Changed bricks to bricks

DariusV72 committed:
Deleted unnecessary constants.dart

DariusV72 committed:
Resolved important issue in pubspec

RobertoWTBW committed:
Delete Bricks.png

RobertoWTBW committed:
Create bricks.png
